packages/dev/codemods/src/s1-to-s2/__tests__/__snapshots__/tabs.test.ts.snap

```diff
@@ -1,8 +1,105 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Renames variants 1`] = `
-"// import {Button} from '@adobe/react-spectrum';
+exports[`Move items from Tabs to TabList 1`] = `
+"import { TabPanels, Item } from '@adobe/react-spectrum';
 
+import { Tabs, TabList } from "@react-spectrum/s2";
+
+let items = [
+  {name: 'Tab 1', children: 'Tab Body 1'},
+  {name: 'Tab 2', children: 'Tab Body 2'},
+  {name: 'Tab 3', children: 'Tab Body 3'},
+  {name: 'Tab 4', children: 'Tab Body 4'},
+  {name: 'Tab 5', children: 'Tab Body 5'},
+  {name: 'Tab 6', children: 'Tab Body 6'}
+];
+
+<Tabs aria-label="Dynamic tabs"><TabList items={tabs}>
+    {(item) => (
+      <Item key={item.name}>
+        {item.name}
+      </Item>
+    )}
+  </TabList></Tabs>"
+`;
+
+exports[`Remove isEmphasized 1`] = `
+"import { Tab, TabPanel, Tabs, TabList } from "@react-spectrum/s2";
+import { TabPanels, Item } from '@adobe/react-spectrum';
+let isEmphasized = true;
+let props = {isEmphasized: true};
 <div>
+  <Tabs aria-label="History of Ancient Rome"><TabList>
+      <Tab id="FoR">Founding of Rome</Tab>
+      <Tab id="MaR">Monarchy and Republic</Tab>
+      <Tab id="Emp">Empire</Tab>
+    </TabList><TabPanel id="FoR">Arma virumque cano, Troiae qui primus ab oris.</TabPanel><TabPanel id="MaR">Senatus Populusque Romanus.</TabPanel><TabPanel id="Emp">Alea jacta est.</TabPanel></Tabs>
```

Comment:
would be nice if the formatting was nicer but I guess they can run prettier or something

Reply:
Since the node gets moved, not sure there's much we can do about this at the AST-level. Hopefully people are running their formatter+linter afterwards anyway